command/aarelayer/params.go

```diff
@@ -0,0 +1,25 @@
+package aarelayer
+
+import (
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command/helper"
+)
+
+const (
+	addrFlag = "addr"
+
+	defaultPort = 8198
```

Comment:
Why this port selection?

Reply:
We can use any other port. Suggestions?

Reply:
Changed to `8081` (`80` and `8080` are standard web server ports, I think we should avoid them)  https://github.com/0xPolygon/polygon-edge/pull/1252/commits/f8527534f60da34f423360164a5bfd3a2cbf29d9

Reply:
Sure, and 8198 is ok! I just wanted to know if you checked something like https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers 8198 look good, 8081 too crowded

Reply:
Reverted back to `8198` https://github.com/0xPolygon/polygon-edge/pull/1252/commits/396931ce5428a8d78bb2bba0bfa8da056edc7fcd
I did not check that list, but thanks, great resource!